blog/_posts/2022-04-04-scala-3.1.2-released.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: blog-detail
+post-type: blog
+by: Paweł Marks, VirtusLab
+title: Scala 3.1.2 released!
+---
+
+We are happy to announce the release of Scala 3.1.2. This new release brings a few significant improvements awaited by the community.
+
+## Configurable Scala output version
```

Comment:
At this point the term `scala output version` might not yet be well understood as its origin is explained in the description of compiler flags. So how about renaming this section to something like `Forward compatible compilation`?

Reply:
Now the first paragraph explains what `configurable scala output means`. Is it better?